package.json

```diff
@@ -109,10 +114,11 @@
     "power-assert": "^1.4.4",
     "prettier": "^1.7.0",
     "proxyquire": "^2.0.0",
+    "query-string": "^6.4.0",
```

Comment:
This is used in `file.ts`, but looks to only be added as a dev dependency.  Also - I wonder out loud if we really need a third party module for this :)